### DISCUSSION TITLE
Where should I Put detection: DetectionService = compre_face.init_face_detection(DETECTION_API_KEY)  in Flask?

### DISCUSSION THREAD

**Discussion by martinenkoEduard** [exadel-inc/CompreFace#830] [Q&A] [answered]
Where should I Put detection: DetectionService = compre_face.init_face_detection(DETECTION_API_KEY)  in Flask?
Should I initialize DetectionService  for each request? 
or should i put it near app = Flask(__name__)

**Comment by pospielov** [comment, marked as answer]
It's better to init CompreFace in one place once and then use those objects everywhere you need them.
